packages/components/tests/dummy/app/templates/components/badge.hbs

```diff
@@ -102,6 +102,23 @@
       <Hds::Badge @icon="activity" @text="Lorem ipsum" /></p>
   </div>
 
+  <h5 class="dummy-h5">Containers</h5>
+  <div class="dummy-form-text-input-containers">
+    {{#let (array "block" "flex" "grid") as |displays|}}
+      {{#each displays as |display|}}
+        <div>
+          <span class="dummy-text-small">Parent with <code class="dummy-code">display: {{display}}</code></span>
+          <br />
+          <div class="dummy-form-text-input-containers__{{display}}">
```

Comment:
same here.

also, better to add some gap between the child elements, otherwise people may think there is some error in the implementation.